classes/kernel-uimage-meson.bbclass

```diff
@@ -11,7 +11,14 @@ python __anonymous () {
 
 do_uboot_mkimage() {
 	if test "x${KERNEL_IMAGETYPE}" = "xuImage" ; then
-		uboot-mkimage -A ${UBOOT_ARCH} -O linux -T kernel -C none -a ${UBOOT_LOADADDRESS} -e ${UBOOT_ENTRYSYMBOL} -n "${DISTRO_NAME}/${PV}/${MACHINE}" -d ${B}/arch/${ARCH}/boot/Image ${B}/arch/${ARCH}/boot/uImage
+
```

Comment:
Hi, 

Thanks for the fix, I planned to do it, but you were faster.

Indeed UBOOT_ENTRYSYMBOL is the wrong one... (my bad), and should be UBOOT_ENTRYPOINT instead, so you can drop the code you added and simply remplace the buggy UBOOT_ENTRYSYMBOL by UBOOT_ENTRYPOINT in the mkimage command